libindy/src/utils/crypto/base64/rust_base64.rs

```diff
@@ -3,11 +3,11 @@ extern crate base64;
 use errors::common::CommonError;
 
 pub fn encode(doc: &[u8]) -> String {
-    base64::encode(doc)
+    base64::encode_config(doc, base64::URL_SAFE)
```

Comment:
As far as I know this `URL_SAFE` alphabet is different for base64 against default one. In this case we must not to change this utility function behavior as it widely used in codebase

Reply:
fixed this by adding `encode_urlsafe()` and `decode_urlsafe()` functions and bringing the old implementations back.